docs/how_to_guides/calculating_elterngeld.ipynb

```diff
@@ -0,0 +1,211 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to calculate Elterngeld\n",
+    "\n",
+    "Elterngeld (parental leave benefit) is a financial support for parents in the first\n",
+    "months after the birth of a child. It replaces some share of the income before birth\n",
+    "(usually about 67% of net income). This net income is approximated using simplified\n",
+    "calculation rules. This document explains how to calculate Elterngeld\n",
+    "based on historical income data for a given household.\n",
+    "\n",
+    "In principle, one can compute Elterngeld in three steps:\n",
+    "1. Compute the average monthly gross income before birth in the data.\n",
+    "2. Call GETTSIM with the target `elterngeld_nettolohn_approximation_m`.\n",
+    "3. Call GETTSIM with the target `elterngeld_m` using the outcome of step 2 as the input\n",
+    "   for `elterngeld_nettoeinkommen_vorjahr_m`.\n",
+    "\n",
+    "In the following, we will explain some more details."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example\n",
+    "\n",
+    "In the following, we compute Elterngeld for a child that is born in July 2024. The\n",
+    "parent that receives Elterngeld had a monthly gross wage of 2000€ in the 12 months\n",
+    "before birth of the child.\n",
+    "\n",
+    "### Step 1: Compute the average monthly gross income before birth\n",
+    "\n",
+    "If income from dependent employment is the only source of income, the basis for the\n",
+    "Elterngeld calculation is the average monthly gross income in the 12 months before the\n",
+    "birth of a child. In our case, this would be July 2023 until June 2024. If other income\n",
+    "sources exist, the basis is the average monthly gross income in the calendar year before\n",
+    "the birth of the child, 2023 in our example. We will stick to the first case here\n",
+    "because:\n",
+    "\n",
+    "> **Note**: The Elterngeld implementation in GETTSIM does currently not consider income\n",
+    "> from self-employment or other sources. See\n",
+    "> [this](https://github.com/iza-institute-of-labor-economics/gettsim/issues/613) issue.\n",
+    "\n",
+    "So let's stick to the case where income from dependent employment is the only source of\n",
+    "income. You can compute this directly if you have monthly information on gross income\n",
```

Comment:
What is meant by "this"? And to the extent it's gross income, I would not need to calculate anything?

Reply:
I guess this comes from me being used to DRV data (where you need to compute this based on Entgeltpunkte) 😄 
I'll adjust that.